static/js/compose_actions.js

```diff
@@ -63,17 +64,34 @@ function get_focus_area(msg_type, opts) {
     }
 
     if (msg_type === "stream") {
-        return "#stream_message_recipient_stream";
+        return "#compose_select_stream_widget";
     }
     return "#private_message_recipient";
 }
 
 // Export for testing
 export const _get_focus_area = get_focus_area;
 
+export function open_compose_stream_dropup() {
+    if ($("#id_compose_select_stream").hasClass("open")) {
+        return;
+    }
+    $("#compose_select_stream_widget").trigger("focus");
+    $("#id_compose_select_stream").addClass("open");
+    $("#id_compose_select_stream input").trigger("focus");
```

Comment:
I couldn't get the dropdown() from bootstrap to work (though I didn't spend that long on it), and this feels more clear to me

Reply:
`$("#id_compose_select_stream > .dropdown-toggle").dropdown("toggle")` works for me.

Reply:
🤷 okay cool! thanks :)

Reply:
switched to that!